src/main/java/de/zalando/ep/zalenium/proxy/AutoStartProxySet.java

```diff
@@ -182,6 +182,16 @@ public void add(RemoteProxy proxy) {
         }
     }
 
+    @Override
+    public void verifyAbilityToHandleDesiredCapabilities(Map<String, Object> desiredCapabilities) {
+        // Using the matcher to see if docker-selenium can handle the desired capabilities. If matches is because the

```

Comment:
Is this comment right? If it matches isn’t it because docker-selenium can handle it? Not the cloud proxies?

Reply:
Ahh I just read that matcher.  Comment is right.